reverse_proxy/meta/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+galaxy_info:
+  author: Aaron Jackson
+  namespace: hic
+  description: Configures nginx as a reverse proxy for accessing services ingress only
+  company: Health Informatics Centre, University of Dundee
+
+  license: MIT
+
+  min_ansible_version: "2.1"
```

Comment:
This is fine, though feel free to bump this to whatever version we're using so that if someone tries to use this with an older version of Ansible they can't complain.